clar2wasm/src/wasm_generator.rs

```diff
@@ -1476,7 +1490,7 @@ impl ASTVisitor for WasmGenerator<'_> {
 
         // Host interface fills the result into the specified memory. Read it
         // back out, and place the value on the data stack.
-        self.read_from_memory(builder.borrow_mut(), offset, &ty);
+        self.read_from_memory(builder.borrow_mut(), offset, 0, &ty);
```

Comment:
I actually never noticed before that we use `borrow_mut()` everywhere.
We should replace them all with regular `&mut`.

Reply:
Yeah, there was some reason for that at some point... 🤷‍♂️ . Let's do that in a future PR.